<launch>
  <node pkg="lane_keep_system" type="lanelet_map_handler" name="lanelet_map_handler" output="screen">
    <param name="lanelet2_path" value="$(find lane_keep_system)/laneletMaps/uni_jkk.osm" />
    <param name="lanelet_frame" value="map" />
    <param name="ego_frame" value="test" />
    <param name="scenario_length" value="80" />
    <param name="polyline_count" value="2" />
    <param name="visualize_path" value="true" />
  </node>
</launch>
